content/post/abortion-bill.en.md

```diff
@@ -0,0 +1,83 @@
+
+In this context, many american women started deleting their apps and requesting past data erasure. But who can certify that past data is fully erased and impossible to undelete? Some women started to use the Clue app that is based in E.U. (Berlin) thinking that GDPR is protecting them. Unfortunately, GDPR here is not really a protection because of what we already flagged above: firstly even a E.U. company has to respond positively to a legitimate legal request from US authorities, secondly, the storage of an E.U. app may not be simple and may use servers located outside Europe. Here again there is a big risk that state police may request this data.
+
+Some startup companies saw in this situation the possibility to make more money. The Stardust app saw its installs increase by 6,000% over one weekend after announcing that it would protect user data with end-to-end encryption. The reality was that the company has yet to implement these measures and has several privacy shortcomings (like sharing mobile numbers with a third-party analytics company)…
+
+Reading privacy policies of those apps is tedious, they can change frequently, and ultimately they can be deemed illegal by state authorities.
+
+Finally a good solution may only be to use an app that processes and stores this sensible data locally (in the smartphone) like Apple Health. But again, a change in the app code could make this strategy evolve negatively regarding privacy. Any full open source app could be the only less-risky solution. Except Euki and Mozilla Foundation-backed Drip app, all others are storing user health data in the cloud. Some apps only share anonymized data (stripped of identifying user information) with third parties; but some techniques like machine learning can allow re-identification of these data. Most apps may share information in order to “comply with or respond to law enforcement”. The extraterritoriality of U.S. laws could lead, then, some non-U.S. apps to be obliged to share their data with state police. Finally, and this is obvious, a good data is a data that is not shared (or collected !).
+
+A low-tech solution using a thermometer, a calendar, and a pencil remains the safest even if it is less attractive.
+
+The FTC sued Kochava on Monday, August 29, 2022 for selling the geolocation information of "hundreds of thousands" of cell phones to anti-abortion groups.  These groups have used geolocation data to identify women who have visited abortion clinics[^10]. The worst future is already now!
+
+#### Shop and e-shops
+
+Some users flag that shops/e-shops that store your buying history may also be a risk. Your purchase history of menstrual hygiene products can also be requested by state police to reveal an illegal abortion. Some propose to desynchronize your purchases of menstrual hygiene products, pay in cash, and cancel your loyalty card (because it is identifying the buyer).
```

Comment:
Plural on loyalty cards? Not sure about it though.

Reply:
I'd say there is only 1 loyalty card per shop so I think it goes. Both would probably be ok.